### PR TITLE
[NOTASK]: Update Directors section grid

### PR DESCRIPTION
**Pull request by FevenSeyfu**
Refactor the about page > Directors card to use grid, changes include

- updated layout's inline padding unit to % to resize on smaller screens
- change the Directors Profile card to Grid
- made changes to the quote container padding and margins to use relative units

### CODE REVIEW

**Comment by joanwolk** [comment]
I think https://github.com/poulami-saha/Global-Code-Queens-MyTeam-Website/pull/41/commits/224978bae87c4cdfc70d6275bcecedc90104c147 might need to be reversed, because proportional padding does stuff like this (680px-767px, wide end of mobile width; the text pushes much too far down the card and the social links get hidden). 
Screenshot 2024-07-31 at 2 15 22 PM

A similar thing can happen at very narrow desktop sizes:
Screenshot 2024-07-31 at 2 18 19 PM

**Comment by joanwolk** [comment]
Using the original padding has its own problems, I see. I'm working on an alternative that uses grid for the profile cards, but I'm going to have to either (1) do something a little hacky with the display to make the card size stretch vertically based on the quote while displaying the avatar, or (2) animate a transition of some kind to account for the card rendering bigger on the quote side. 

Alternative solutions:
- we leave the directors section as it was, which will, in fact, be 3 columns wide at 1440px and above
- we leave it as it was, BUT add some negative margins to the `.profiles` div so it gets up to 3 columns sooner. How weird this looks will depend on how much vertical space the user has to be able to see the profile cards extend past the header/footer edges; example below with `margin-inline: -60px;` at viewport width of 1322. It's noticeable but maybe ok?
Screenshot 2024-07-31 at 4 16 58 PM

**Comment by FevenSeyfu** [comment]
Hi @joanwolk  as suggested I have reverted the changes to make the padding relative. I see the options you have listed and the issue I had with making the cards vertically responsive was the layout of the cards and the buttons being aligned relative to the card without a fixed height for the cards it misaligned it for Quote vs the avatar containers.                      
As an alternative review, my latest commit is not an ideal solution but considering the time keeping the inline padding relative and moving the breaking point for the desktop from 960px to 1250 or even 1440px for the grid it should keep both quotes and avatar cards correctly aligned.   

- or our other option is keeping it as is, the issue I have is with the flex-wrap for bigger desktops it keeps wrapping cards which causes it to lose form we can also limit that in some way 
image

**Review by eunicey — COMMENTED** [review]
From what I'm understanding, with larger desktops we either lose the 3x2 structure of the cards (if we keep it as it was) or the button overlaps with the social links (with the latest commit). I would rather lose the structure than have the content overlap, IMO.

**Comment by joanwolk** [comment]
> * our other option is keeping it as is, the issue I have is with the flex-wrap for bigger desktops it keeps wrapping cards which causes it to lose form we can also limit that in some way

This is easily fixed by giving a max-width to the `.profiles` div so it can't get wider than the 3-column layout

File: src/components/AboutContainer/Directors/Directors.module.css [hunk 1]
```diff
@@ -46,17 +45,20 @@
...
 
   .profiles {
+    grid-template-columns: repeat(2, 1fr);
     row-gap: 3rem;
+    column-gap: 0.688rem;
   }
 }
 
-@media (min-width: 960px) {
+@media (min-width: 1250px) {
```
Comment (poulami-saha) [inline src/components/AboutContainer/Directors/Directors.module.css:54]: I think we are getting deviated a bit from the requirement now, we finalized only 2 media breakpoints 768 and 960, I believe for the rest of the application 1250 is not being considered

Comment (FevenSeyfu) [reply src/components/AboutContainer/Directors/Directors.module.css:54]: Yes that is why I have mentioned it is not and ideal solution considering the time I can't redo the profileCards to be vertically responsive in time if anyone else can feel free to do so, as suggested we can leave it as is and  close the PR  @eunicey  can you redo your changes in separate PR or revert the UI changes and merge it in this PR.

Comment (eunicey) [reply src/components/AboutContainer/Directors/Directors.module.css:54]: @FevenSeyfu - I don't know what the final consensus is on the UI changes here so I'll leave this PR alone and make a separate PR for the testing scripts.

**Review by joanwolk — CHANGES_REQUESTED** [review]
Please review #45 and if that resolves the issue well enough, that one can be accepted and this can be closed WITHOUT merging.

**Pull request closed without merging**